### PR TITLE
add $\sigma$-connected and theorems

### PR DESCRIPTION
**Pull request by Jianing-Song**
This is a retry of https://github.com/pi-base/data/pull/610.

### CODE REVIEW

**Comment by prabau** [comment]
P189: I am sure I understand the characterization in terms of continuous maps into S15 (= cofinite topology on $\omega$).  (did you mean $omega$ with the discrete topology by any chance?)  It is clear that if $X$ is $\sigma$-connected, any map from $X$ to a countable T1 space is constant.  But if $X$ is not $sigma$-connected, why is there a continuous map into ...? Is that even true?

**Comment by prabau** [comment]
On further examination, it is indeed easy to see.  No problem there.

**Comment by prabau** [comment]
Looks like it has all the main ideas.  Here are some more detailed comments, some nitpicking but if pi-base is going to serve as a reference for the future, we might as well write it as good as possible.

P189:
- Since the main name $\sigma$-connected is used in the literature and well-known by people in that area, I don't think the alias is adding anything and I would remove it.
- The phrase "disjoint union" could be misleading to people who are used to "topological disjoint union" (= coproduct).  And the phrasing is also slightly hard to parse maybe.  How about something more explicit, similar to Nadler:
> "The space cannot be partitioned into more than one and at most countably infinitely many nonempty closed sets"

- We could then add another paragraph: (one benefit of this: T484 becomes trivial)
> "Equivalently, $X$ is connected and cannot be written as a countably infinite union of pairwise disjoint nonempty closed sets."

- First paragraph: the second sentence rephrasing things in terms of the $C_n$ really seems redundant. If it does not add anything, does it need to be there?
- Second paragraph: typos: "Equivalently", "continuous"
- Second paragraph: Can we remove the justification?  If one thinks about it, it's easy to check.  But if we need to spell it out, what you wrote is only one half of it; we would also have to show the reverse (i.e. not sigma connected ==> there is a non-constant map ...).  Also, it's not strictly the case that a partition ... is "equivalent to" ..., etc.  It seems just easier and simpler to remove that justification altogether.

T484: "By definition" and remove all the rest.

T485:
- Actually, Sierpinski worked specifically in the context of subsets of Euclidean space:
(see https://www.jstage.jst.go.jp/article/tmj1911/13/0/13_0_300/_article/-char/ja/) But it's probably fine.
- line 4: remove `and:` and remove the leading hyphen on line 5

T487: could we say something like "The space is the countable union of singletons which are closed and pairwise disjoint" ?

**Comment by Jianing-Song** [comment]
> Looks like it has all the main ideas. Here are some more detailed comments, some nitpicking but if pi-base is going to serve as a reference for the future, we might as well write it as good as possible.
> 
> P189:
> 
> * Since the main name σ-connected is used in the literature and well-known by people in that area, I don't think the alias is adding anything and I would remove it.
> * The phrase "disjoint union" could be misleading to people who are used to "topological disjoint union" (= coproduct).  And the phrasing is also slightly hard to parse maybe.  How about something more explicit, similar to Nadler:
> 
> > "The space cannot be partitioned into more than one and at most countably infinitely many nonempty closed sets"
> 
> * We could then add another paragraph: (one benefit of this: T484 becomes trivial)
> 
> > "Equivalently, X is connected and cannot be written as a countably infinite union of pairwise disjoint nonempty closed sets."
> 
> * First paragraph: the second sentence rephrasing things in terms of the Cn really seems redundant. If it does not add anything, does it need to be there?
> * Second paragraph: typos: "Equivalently", "continuous"
> * Second paragraph: Can we remove the justification?  If one thinks about it, it's easy to check.  But if we need to spell it out, what you wrote is only one half of it; we would also have to show the reverse (i.e. not sigma connected ==> there is a non-constant map ...).  Also, it's not strictly the case that a partition ... is "equivalent to" ..., etc.  It seems just easier and simpler to remove that justification altogether.
> 
> T484: "By definition" and remove all the rest.
> 
> T485:
> 
> * Actually, Sierpinski worked specifically in the context of subsets of Euclidean space:
>   (see https://www.jstage.jst.go.jp/article/tmj1911/13/0/13_0_300/_article/-char/ja/) But it's probably fine.
> * line 4: remove `and:` and remove the leading hyphen on line 5
> 
> T487: could we say something like "The space is the countable union of singletons, which are closed and pairwise disjoint" ?

Thanks for your suggestions! I've done as you said. How would we cite the original article of Sierpinski?

**Comment by Jianing-Song** [comment]
> Looks like it has all the main ideas. Here are some more detailed comments, some nitpicking but if pi-base is going to serve as a reference for the future, we might as well write it as good as possible.
> 
> P189:
> 
> * Since the main name σ-connected is used in the literature and well-known by people in that area, I don't think the alias is adding anything and I would remove it.
> * The phrase "disjoint union" could be misleading to people who are used to "topological disjoint union" (= coproduct).  And the phrasing is also slightly hard to parse maybe.  How about something more explicit, similar to Nadler:
> 
> > "The space cannot be partitioned into more than one and at most countably infinitely many nonempty closed sets"
> 
> * We could then add another paragraph: (one benefit of this: T484 becomes trivial)
> 
> > "Equivalently, X is connected and cannot be written as a countably infinite union of pairwise disjoint nonempty closed sets."
> 
> * First paragraph: the second sentence rephrasing things in terms of the Cn really seems redundant. If it does not add anything, does it need to be there?
> * Second paragraph: typos: "Equivalently", "continuous"
> * Second paragraph: Can we remove the justification?  If one thinks about it, it's easy to check.  But if we need to spell it out, what you wrote is only one half of it; we would also have to show the reverse (i.e. not sigma connected ==> there is a non-constant map ...).  Also, it's not strictly the case that a partition ... is "equivalent to" ..., etc.  It seems just easier and simpler to remove that justification altogether.
> 
> T484: "By definition" and remove all the rest.
> 
> T485:
> 
> * Actually, Sierpinski worked specifically in the context of subsets of Euclidean space:
>   (see https://www.jstage.jst.go.jp/article/tmj1911/13/0/13_0_300/_article/-char/ja/) But it's probably fine.
> * line 4: remove `and:` and remove the leading hyphen on line 5
> 
> T487: could we say something like "The space is the countable union of singletons, which are closed and pairwise disjoint" ?

Just a little question: You would prefer Sierpiński or Sierpinski (with no "ń"), or both are OK? I see that the space S10 is called "Sierpinski Space".

**Comment by Moniker1998** [comment]
I think both Sierpiński and Sierpinski are okay, the English keyboard doesn't have ń on it. I usually write Sierpiński but that's only because I'm from Poland myself

**Comment by prabau** [comment]
> You would prefer Sierpiński or Sierpinski (with no "ń"), or both are OK? I see that the space S10 is called "Sierpinski Space".

It does not matter too much.  Sierpinski is easier to type, but Sierpiński is the Polish spelling and what he was using in his papers.  It seems that pi-base is not too concerned with this, but if needed, we can go with Sierpiński.

**Comment by StevenClontz** [comment]
I've written Lindelöf enough times recently to make this page (that others can contribute to): https://github.com/pi-base/data/wiki/Special-Characters

**Comment by Jianing-Song** [comment]
Thank you for you all for your responses! Please kindly check if everything goes well :)

File: properties/P000189.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+uid: P000189
+name: $\sigma$-connected
+refs:
+  - doi: 10.1201/9781315274089
+    name: Continuum Theory
```
Comment (prabau) [inline properties/P000189.md:6]: For books and papers, we usually add the author in parentheses.  I.e., change it to `name: Continuum Theory (Nadler)`.  Makes it easier to figure out what things are when looking at the web page.

**Comment by prabau** [comment]
P189: third paragraph.  As discussed already, the justification you have for the characterization by maps into S15 is incomplete.  But it seems to me it would be good enough to leave the justification to the reader, as it is pretty easy to check.  Would be simpler and easier to read if we leave that out.  Do have any comment on that?

File: properties/P000189.md (hunk 1 above)
Comment (StevenClontz) [reply properties/P000189.md:6]:
```suggestion
    name: Continuum Theory (Nadler)
```

File: theorems/T000487.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+uid: T000487
+if:
+  P000057: true
+  P000002: true
+  P000125: true
```
Comment (StevenClontz) [inline theorems/T000487.md:6]:
```suggestion
if:
  and:
    - P000057: true
    - P000002: true
    - P000125: true
```

Should fix the build error.

**Comment by prabau** [comment]
@Jianing-Song You can now preview the results on https://topology.pi-base.org by clicking on the Advanced tab and entering `sigma-connected` for the branch.

**Comment by Jianing-Song** [comment]
> P189: third paragraph. As discussed already, the justification you have for the characterization by maps into S15 is incomplete. But it seems to me it would be good enough to leave the justification to the reader, as it is pretty easy to check. Would be simpler and easier to read if we leave that out. Do have any comment on that?

I edited P189 accordingly. I was not sure about what had happened with the "Merge branch 'pi-base:main' into sigma_connected" commit, because when I had seen that my branch was 2 commits behind pi-base/data:main I clicked Sync fork, but this operation seemed to be unnecessary :)

**Comment by prabau** [comment]
@Jianing-Song Sorry, I think the characterization in terms maps into S25 was good and nice to have.   But I thought its justification was slightly misleading.  Either keep the characterization without explicit justification or we change it slightly.  Maybe something like this:

> Equivalently, every continuous map from $X$ to {S15} is constant.  (This is easy to check using the fact that the codomain is countable and $T_1$, and each of its proper closed subsets is a finite union of singletons.)

If you prefer not to have it for now, we can add it later.  Just let us know and we can approve this PR.

**Comment by prabau** [comment]
> I was not sure about what had happened with the "Merge branch 'pi-base:main' into sigma_connected" commit, because when I had seen that my branch was 2 commits behind pi-base/data:main I clicked Sync fork, but this operation seemed to be unnecessary :)

FYI,  in general it's perfectly fine to be a few commits behind main.  That does not prevent merging later on, as long as there are no conflicts.  (@StevenClontz do you agree?)

**Comment by Jianing-Song** [comment]
> @Jianing-Song Sorry, I think the characterization in terms maps into S25 was good and nice to have. But I thought its justification was slightly misleading. Either keep the characterization without explicit justification or change the justification slightly. Maybe something like this:
> 
> > Equivalently, every continuous map from X to {S15} is constant.  (This is easy to check using the fact that the codomain is countable and T1, and each of its proper closed subsets is a finite union of singletons.)
> 
> If you prefer not to have it for now, we can add it later. Just let us know and we can approve this PR.

Thanks, I like that sentence! So I added it :)

**Comment by prabau** [comment]
@Jianing-Song Approving.  Sorry it took so long and all the nitpicking.  But it's a good to have the property now in pi-base.  We really appreciate your contribution.  It would be great if you want to contribute more things in the future (for example, spaces in the plane that are connected but not sigma-connected, like Mullikin's nautilus (mentioned in https://math.stackexchange.com/questions/4883752), or other things).  By the way, there is not much more complication to adding a space versus a property or theorems.

**Review by prabau — APPROVED** [review]

**Comment by StevenClontz** [comment]
Hi @Jianing-Song, thanks for this contribution! You've been invited to be a maintainer of this repository, which means you can create branches directly on here rather than your fork, and they'll be automatically compiled and pushed to the internet (so you can preview them at topology.pi-base.org). You can also participate by reviewing other pull requests as well. Let me know if you have any questions. :-)

**Comment by Jianing-Song** [comment]
> Hi @Jianing-Song, thanks for this contribution! You've been invited to be a maintainer of this repository, which means you can create branches directly on here rather than your fork, and they'll be automatically compiled and pushed to the internet (so you can preview them at topology.pi-base.org). You can also participate by reviewing other pull requests as well. Let me know if you have any questions. :-)

Thanks! In that way I could plan to add the example in Continuum theory (perhaps in the next month) :)
